workbench/package.json

```diff
@@ -15,11 +15,10 @@
     "test-electron-app": "jest --runInBand --testMatch **/tests/binary_tests/*.test.js",
     "test-sampledata-registry": "jest --runInBand --testMatch **/tests/sampledata_linkcheck/*.test.js",
     "postinstall": "electron-builder install-app-deps",
-    "build-main": "babel --delete-dir-on-start src/main -d build/main --verbose --config-file ./babel.config.js --ignore **/__mocks__/* -s both",
+    "build-main": "babel --delete-dir-on-start src/main -d build/main --verbose --config-file ./babel.config.js --ignore **/__mocks__/* -s both --copy-files",
```

Comment:
Copies the json files over to the build dir

Reply:
Unfortunately it looks like we also need to add `--no-copy-ignored` for reasons: https://github.com/babel/babel/issues/6226

Jest was warning us about a duplicated mock because it found `__mocks__/logger.js` in `build/` and in `src/`.